docs/herramientas/plantilla-readme.md

```diff
@@ -7,7 +7,7 @@ sidebar_label: Plantilla Readme
 README es un archivo que trae una pequeña descripción sobre todo el proyecto, que contiene una introducción y explicación de este. El tipo de información que se puede describir aquí es información solicitada para comprender el proyecto y su estructura o propósito. A continuación, brindamos una plantilla para crear un archivo README para un nuevo proyecto.
 
 
-### Para Copiar esta Plantilla Readme como Código Fuente Markdown [haga clic aquí](plantilla-readme-codigo-fuente)
+### Para Copiar esta Plantilla Readme como Código Fuente Markdown [haga clic aquí](./plantilla-readme-codigo-fuente.md)
```

Comment:
i believe this should reference the page id and not the markdown file name as once the site is compiled that markdown file is no longer used